web/client/plugins/querypanel/CrossLayerFilter.jsx

```diff
@@ -0,0 +1,193 @@
+/*
```

Comment:
This is a connected copy of `web/client/components/data/query/CrossLayerFilter.jsx` .
Please reuse the component.